src/Execution/Execution.php

```diff
@@ -3,34 +3,29 @@
 namespace Digia\GraphQL\Execution;
 
 use Digia\GraphQL\Error\ErrorHandlerInterface;
+use Digia\GraphQL\Execution\Strategy\FieldCollector;
+use Digia\GraphQL\Execution\Strategy\ParallelExecutionStrategy;
+use Digia\GraphQL\Execution\Strategy\SerialExecutionStrategy;
 use Digia\GraphQL\Language\Node\DocumentNode;
 use Digia\GraphQL\Language\Node\FragmentDefinitionNode;
 use Digia\GraphQL\Language\Node\FragmentSpreadNode;
 use Digia\GraphQL\Language\Node\OperationDefinitionNode;
 use Digia\GraphQL\Schema\Schema;
+use React\Promise\PromiseInterface;
 
 class Execution implements ExecutionInterface
 {
-
     /**
-     * @param Schema        $schema
-     * @param DocumentNode  $documentNode
-     * @param mixed         $rootValue
-     * @param mixed         $contextValue
-     * @param array         $variableValues
-     * @param null|string   $operationName
-     * @param callable|null $fieldResolver
-     * @return ExecutionResult
-     * @throws \Throwable
+     * @inheritdoc
      */
     public function execute(
         Schema $schema,
         DocumentNode $documentNode,
         $rootValue = null,
         $contextValue = null,
         array $variableValues = [],
-        ?string $operationName = null,
-        ?callable $fieldResolver = null,
+        string $operationName = null,
+        callable $fieldResolver = null,
```

Comment:
These should not have been changed, if you can accept null then it was to be `?foo`

Reply:
This is now fixed.